docs/scos/dev/glue-api-guides/202204.0/glue-backend-api/how-to-guides/how-to-create-grant-type-parameters.md

```diff
@@ -0,0 +1,134 @@
+---
+title: How to create grant type parameters
+description: This document explains how to create the grant type parameters and how to use them in code
+last_updated: October 24, 2022
+template: howto-guide-template
+redirect_from:
+  - /docs/scos/dev/glue-api-guides/202204.0/glue-backend-api/how-to-guides/create-grant-type-parameters.html
+---
+
+This document explains how to create the grant type parameter and use it.
+
+Integrate authentication following the [Glue API Authentication integration](/docs/scos/dev/technical-enhancement-integration-guides/glue-authentication-integration.html) guide.
+
+Glue provides grant types `password` for a customer, and a user out of the box:
```

Comment:
coma is not needed if we have only one `and` as a I know

Reply:
Also please add a link to https://www.rfc-editor.org/rfc/rfc6749#appendix-A.10

Reply:
>coma is not needed if we have only one and as a I know
yep, you're right and I thought the same, it was fixed because Grammarly for some reason pointed out that 🤷‍♀️